src/google_takeout_metadata/exif_writer.py

```
@@ -15,370 +20,166 @@ def _fmt_dt(ts: int | None, use_localtime: bool) -> str | None:
-
+    
+    # Ajouter les arguments métadonnées
+    cmd.extend(args)
+    # Ajouter le fichier à traiter
+    cmd.append(str(media_path))
+    
+    print(f"DEBUG: Commande complète: {cmd}")
+    
     try:
-        subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=60, encoding='utf-8')
+        result = subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=60, encoding='utf-8')
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)